### DISCUSSION TITLE
Plotting cycle

### DISCUSSION THREAD

**Discussion by leonidcan** [ericaltendorf/plotman#573] [Q&A]
I am confused how to use this.
If I run interactive and my session disconnects, will plotman stop scheduling new jobs?
If I enter in plotting cycle in background I guess it will continue to work even I disconnect terminal, correct?
Now if I have active plotting cycle, can I still use interactive mode or there will be conflict?

**Comment by leonidcan** [comment]
Anyone can answer? For now I only use interactive but if I close terminal will it stop scheduling?

**Reply by jayhohoho2019** [reply]
yes it will stop starting new jobs if you stop the interactive session, which imho is a pain.

**Reply by Purple-Tentacle** [reply]
You could use screen or tmux. That way the session stays alive when you close the terminal or ssh session.
